### ISSUE TITLE
Rule to decide if PROPN or NOUN (Diseases, Hormones etc.)

### ISSUE THREAD

**Issue by yaelFinkelshtein** [IAHLT/UD_Hebrew#15]
Hi, @amir-zeldes 

now we are working on several medical entries. How can we determine if something is PROPN or NOUN?
For example in my files - diabetes, insulin, glucose etc.

Hila suggested maybe according to the written option in English - Capitalized or not. Is it enough? As I see it, even though "diabetes" is not capitalized, it's still a disease's name, and it felt natural to give it PROPN upos.

Thanks

**Comment by amir-zeldes** [comment]
Generally speaking, the 'universal' UD guidelines specify that PROPNs are a "specific individual, place, or object", so I think glucose, insulin etc. are definitely not covered. The same applies to disease names: I think we need to distinguish here between technical *terms* and proper nouns, if that makes sense.

**Comment by NathanD38** [comment]
@amir-zeldes 
Should song, album or book titles remain `NOUN` according to UD guidelines?

> .השיר קיבל מספר נקודות זהה לשירה של חדווה עמרני, בלב אחד

Should **lev** in the song title be `NOUN` or `PROPN`?

If we change it to `PROPN`, should we remove the `Number` and `Gender` features?

**Comment by amir-zeldes** [comment]
Yes, titles of works of art count, and anything that would be NOUN in them should become PROPN. As a result, Gender and Number should be deleted based on our current limitations, though I'm sure that can be done automatically en masse for all data based on POS at the very end in QA if you want to leave it till then.

**Comment by shirawigi** [comment]
@amir-zeldes 
And what about holidays names, such as פסח (which in HTB is PROPN) and ראש השנה (which isn't a PROPN in HTB)?

Also, I think that diseases names should be PROPN, because if we look at the group of diseases which contain cancer, pneumonia, diabetes etc. we have only one entity named diabetes. 
I don't exactly understand how a disease name is a technical term. Could you please explain what did you mean?

**Comment by amir-zeldes** [comment]
Thanks for the holiday example, I just used it in a UD meeting :)

I think there's no specific philosophical reason why diseases are not considered PROPN, I wouldn't be surprised if it's just because Western European languages don't capitalize them. But if you did tag them PROPN, you'd get into a slippery slope (is "coughing" a PROPN? How about "sniffles"? How official does the disease name need to be?). How about breeds of animals - is "I have a poodle" PROPN? What about "the breed Poodle"? We just don't have the luxury of capitals to help us.

I think maybe ideologically there's an idea that names have to be given as names by the population for something that doesn't have a name, meaning something that isn't just available in the language already as a word (so I think having a cold or flu is so 'normal' that no one thinks of it as a name). With things like COVID-19, you would probably see people tagging them as PROPN. For brands it's more obvious: Coca Cola named their drink, and it's therefore a name (but what about "a coke", regardless of brand?)

I think the reason for פסח and ראש השנה in HTB is that the first is not transparently associated with a common noun, whereas the latter is a combination of normal nouns that has a special meaning. Probably it would be tagged PROPN in English (and compare New Year).

My instinct is to avoid this for diseases just because it's not done in other languages, but it's hard to draw a clear line (OntoNotes has an NER category for diseases, although they are tagged as common nouns!)

**Comment by Hilla-Merhav** [comment]
Hi @amir-zeldes

I think my next question is more relevant to a discussion named "Rule to decide if PROPN or ADJ" rather than "PROPN or NOUN", but I felt this issue is close enough to reuse :)

I understood only NOUNs can be tagged PROPN and if the syntax is transparent, deprels should behave as usual, so בנק הפועלים is PROPN+DET+PROPN (and connected through `compound`) and בנק לאומי is PROPN+ADJ (and connected through `amod`).

My batch refers to Clalit Health Services, and despite the fact כללית is originally an ADJ, in my data it usually doesn't behave as an `amod`:

**מחירון כללית** לתרופות ללא מרשם (feels like a `compound`)
קופת חולים **הכללית** (feels like a `compound` because there is not agreement of the definiteness – it's not קופת **ה**חולים **ה**כללית)

Do you think כללית should be tagged PROPN, or should we keep it ADJ that governed through `compound`?

**Comment by amir-zeldes** [comment]
It is `compound` no matter how you look at it, but it could be ADJ if you treat it as ellipsis, or NOUN (and by extension PROPN) if you treat it as a nominalized expression. I think here the second interpretation is justified, as you could have a whole text discussing "klalit" vs. other insurances, and I think that's just the name used for this 'brand'. So I would go with PROPN here.

**Comment by Hilla-Merhav** [comment]
Thanks!
